src/Neo.CLI/CLI/MainService.Logger.cs

```diff
@@ -32,12 +33,12 @@ partial class MainService
 
         private void Initialize_Logger()
         {
-            Utility.Logging += OnLog;
+            Utility.Logging += ((ILoggingHandler)this).Utility_Logging_Handler;
```

Comment:
Why the cast? we need to define in all the partial files the interface `ILoggingHandler`?

Reply:
cause to make the method private, have to implement them implicitly, making it to be used this way.... Otherwise interface will be public.